easybuild/easyconfigs/n/nodejs/nodejs-18.12.1-GCCcore-12.2.0.eb

```
@@ -19,6 +19,7 @@ checksums = ['ba8174dda00d5b90943f37c6a180a1d37c861d91e04a4cb38dc1c0c74981c186']
 builddependencies = [
     ('binutils', '2.39'),
     ('Python', '3.10.8', '-bare'),
+    ('pkgconfig', '1.5.5', '-python'),
```

This should probably just be `pkgconf`, there's no need to pull in the Python bindings to `pkgconfig` (and it results in a dependency conflict with the `bare` Python that is already there as a build dependency):

```suggestion
    ('pkgconf', '1.9.3'),